youcan-shop/cod-theme-3
# DARK - Meraki theme > if the navbar is hidden the sticky express checkout disappear

##  Issue

After discovering the issue that was mentioned in the support slack channel: [link](https://youcan-shop.slack.com/archives/C04BDG9HEB1/p1721060304352489)

I realize that the issue was from some variables that have the values of `null`, so it makes the _**main.js**_ file break and stops the execution of the code each time he faces a nullable variable that was used to do some logic.

## Screenshot of the error
<img width="1790" alt="Screenshot 2024-07-16 at 12 24 51" src="https://github.com/user-attachments/assets/c9597590-29f1-40f1-8f5d-2c3227f4985d">

## QA Steps
-  [ ] Go to the product page and try to hide the navbar
-  [ ] Make sure that the express checkout is **Sticky**
-  [ ] Check if the problem was fixed

## Note

Leave empty when you have nothing to say
